python/smqtk/web/iqr_service/__init__.py

```diff
@@ -626,6 +626,9 @@ def run(self, host=None, port=None, debug=False, **options):
         self.add_url_rule('/session',
                           view_func=self.clean_session,
                           methods=['DELETE'])
+        self.add_url_rule('/adjudicate',
```

Comment:
Yes. There is an adjudicate function above, but there wasn't an end point for it. Unless you can think of a reason why there shouldn't be an endpoint for that function.